sdk/src/connections/usb/usb_depth_sensor.h

```diff
@@ -97,9 +97,9 @@ class UsbDepthSensor : public aditof::DepthSensorInterface {
     initTargetDepthCompute(uint8_t *iniFile, uint16_t iniFileLength,
                            uint8_t *calData, uint16_t calDataLength) override;
     virtual aditof::Status
-    getIniParams(std::map<std::string, float> &params) override;
+    getDepthComputeParams(std::map<std::string, std::string> &params);
     virtual aditof::Status
-    setIniParams(const std::map<std::string, float> &params) override;
+    setDepthComputeParams(const std::map<std::string, std::string> &params);
```

Comment:
The overide specifier is missing from here as well. Please check the other places too.

Reply:
fixed